modules/600-secret-copier/templates/validation.yml

```diff
@@ -0,0 +1,58 @@
+{{- if and (include "helm_lib_kind_exists" (list . "ValidatingAdmissionPolicy")) (include "helm_lib_kind_exists" (list . "ValidatingAdmissionPolicyBinding")) }}
+{{- $policyName := "secret-copier-label.deckhouse.io" }}
+apiVersion:  {{ include "helm_lib_get_api_version_by_kind" (list . "ValidatingAdmissionPolicy") }}
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: {{ $policyName }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "deckhouse") ) | nindent 2 }}
+spec:
+  auditAnnotations:
+    - key: source-user
+      valueExpression: '''User: '' + string(request.userInfo.username) + '' tries to
+      create secret with `secret-copier.deckhouse.io/enabled` label outside of `default` namespace'''
+  failurePolicy: Fail
+  matchConstraints:
+    matchPolicy: Equivalent
+    namespaceSelector:
+      matchExpressions:
```

Comment:
I guess it's better to place this selector in the ValidatingAdmissionPolicyBinding

Reply:
moved namespaceselector to the binding resource